interactions/api/models/user.py

```diff
@@ -63,3 +65,20 @@ def mention(self) -> str:
         :rtype: str
         """
         return f"<@{self.id}>"
+
+    def avatar_url(self, animated: Optional[bool] = True) -> str:
```

Comment:
Please convert this into a property method instead, and build an internal condition to check whether to return an animated representation of the avatar or not.